### DISCUSSION TITLE
Charging wall

### DISCUSSION THREAD

**Discussion by mcharach** [aderusha/DDD-Printable-Wall-Control-System#64] [General]
Hi Allen, 

Thanks for this great project.  I am putting together a battery charging wall in my home and several items like my Dyson stick vac were just designed to be screwed to a wall.  I could go ahead you use some M4 or M5 screws with bolts, but the defeats the entire idea of a wall control system.  Eventually I would have to remove the panel from the wall to make changes to my layout. 

I would like to know if anyone has any good ideas to secure items that are to just be screwed to a wall and also if there are any simple low profile solutions for keyhole mounting items like chargers etc.  I have seen some suggestions regarding automotive Christmas tree fasteners do they work?

https://www.amazon.ca/Akozon-Rivets-Universal-Fastener-Retainer/dp/B07VSPTFLF/ref=asc_df_B07VSPTFLF/?tag=googleshopc0c-20&linkCode=df0&hvadid=335384137364&hvpos=&hvnetw=g&hvrand=8980991796201196074&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9001172&hvtargid=pla-1041921125672&psc=1

**Comment by leviable** [comment]
@mcharach Although I haven't personally used them, I've seen well nuts recommended for exactly what you are describing:

https://www.amazon.com/Rubber-Brass-Insert-Expansion-Sizes/dp/B07KJKNCWX?th=1

**Comment by leviable** [comment]
Also, if well nuts are too weak, you could consider using rivet nuts. That would definitely be stronger, but would be a semi-permanent solution. Semi-permanent in the sense that the rivnuts could be removable with effort, such as drilling them out. Note that if you don't want to buy the tool you can make a rivet nut tool out of a bolt and some nuts (google it), which isn't nearly as convenient but still works.

https://www.amazon.com/RZX-Riveter-Riveting-Setting-mandrels/dp/B07P7M4BKF/ref=sr_1_9?crid=9GXXSY8OBPFF&keywords=rivnut&qid=1674908313&sprefix=rivnu%2Caps%2C166&sr=8-9&th=1

**Comment by razorfrog** [comment]
What about this? [https://www.printables.com/model/206047-wall-control-sturdy-hanger-w-m4m5-screw](https://www.printables.com/model/206047-wall-control-sturdy-hanger-w-m4m5-screw )
